pocs/react-hooks/src/App.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { useSession } from "./App";
+
+describe("App", () => {
+  describe("Session Provider", () => {
+    it("should do stuff", () => {
+      expect(useSession).not.toBeUndefined();
+    });
+  });
+  describe("useSession hook", () => {
+    it("should do stuff", () => {
+      expect(true).toBe(true);
```

Comment:
lol, weird tests.

and why describe inside describe for such a small test?

Reply:
check!